ext/gd/gd.stub.php

```diff
@@ -100,156 +101,157 @@ function imagecreatefrombmp(string $filename): GdImage|false {}
+function imagefilledpolygon(GdImage $image, array $points, int $num_points_or_color, ?int $color = null): bool {}
 
 function imagefontwidth(int $font): int {}
 
 function imagefontheight(int $font): int {}
 
-function imagechar(GdImage $im, int $font, int $x, int $y, string $c, int $col): bool {}
+function imagechar(GdImage $image, int $font, int $x, int $y, string $char, int $color): bool {}
 
-function imagecharup(GdImage $im, int $font, int $x, int $y, string $c, int $col): bool {}
+function imagecharup(GdImage $image, int $font, int $x, int $y, string $char, int $color): bool {}
 
-function imagestring(GdImage $im, int $font, int $x, int $y, string $str, int $col): bool {}
+function imagestring(GdImage $image, int $font, int $x, int $y, string $string, int $color): bool {}
 
-function imagestringup(GdImage $im, int $font, int $x, int $y, string $str, int $col): bool {}
+function imagestringup(GdImage $image, int $font, int $x, int $y, string $string, int $color): bool {}
 
-function imagecopy(GdImage $dst_im, GdImage $src_im, int $dst_x, int $dst_y, int $src_x, int $src_y, int $src_w, int $src_h): bool {}
+function imagecopy(GdImage $dst_image, GdImage $src_image, int $dst_x, int $dst_y, int $src_x, int $src_y, int $src_width, int $src_height): bool {}
 
-function imagecopymerge(GdImage $dst_im, GdImage $src_im, int $dst_x, int $dst_y, int $src_x, int $src_y, int $src_w, int $src_h, int $pct): bool {}
+function imagecopymerge(GdImage $dst_image, GdImage $src_image, int $dst_x, int $dst_y, int $src_x, int $src_y, int $src_width, int $src_height, int $pct): bool {}
 
-function imagecopymergegray(GdImage $dst_im, GdImage $src_im, int $dst_x, int $dst_y, int $src_x, int $src_y, int $src_w, int $src_h, int $pct): bool {}
+function imagecopymergegray(GdImage $dst_image, GdImage $src_image, int $dst_x, int $dst_y, int $src_x, int $src_y, int $src_width, int $src_height, int $pct): bool {}
 
-function imagecopyresized(GdImage $dst_im, GdImage $src_im, int $dst_x, int $dst_y, int $src_x, int $src_y, int $dst_w, int $dst_h, int $src_w, int $src_h): bool {}
+function imagecopyresized(GdImage $dst_image, GdImage $src_image, int $dst_x, int $dst_y, int $src_x, int $src_y, int $dst_width, int $dst_height, int $src_width, int $src_height): bool {}
 
-function imagesx(GdImage $im): int {}
+function imagesx(GdImage $image): int {}
 
-function imagesy(GdImage $im): int {}
+function imagesy(GdImage $image): int {}
 
-function imagesetclip(GdImage $im, int $x1, int $x2, int $y1, int $y2): bool {}
+function imagesetclip(GdImage $image, int $x1, int $x2, int $y1, int $y2): bool {}
 
-function imagegetclip(GdImage $im): array {}
+function imagegetclip(GdImage $image): array {}
 
 #ifdef HAVE_GD_FREETYPE
-function imageftbbox(float $size, float $angle, string $font_file, string $text, array $extrainfo = []): array|false {}
+function imageftbbox(float $size, float $angle, string $font_filename, string $string, array $extra_info = []): array|false {}
```

Comment:
We used additional instead of extra in a couple of places, so maybe we it should be worth to follow this convention here as well. But again, I'm not opposed to use "extra" here.

Reply:
I've renamed this to use `$options`. Doesn't seem like this any different than other option array params.

Reply:
That's even better!